changelogs/unreleased/add-oss-container-doc.yml

```diff
@@ -0,0 +1,4 @@
+---
+description: Added OSS container documentation.
+change-type: patch
+destination-branches: [master]
```

Comment:
should also go in iso4 no? Because otherwise you cannot add an override to the iso4 docs

Reply:
We should if we don't overwrite the files in iso, but we will probably overwrite all of them.  I could do it just to be safe though.

Reply:
Done